docs/data/base/components/switch/switch.md

```diff
@@ -22,23 +21,32 @@ To make the switch accessible, you should ensure the corresponding label reflect
 import SwitchUnstyled from '@mui/base/SwitchUnstyled';
 ```
 
-The `SwitchUnstyled` component provides default components and assigns CSS classes you can style entirely on your own.
-You are free to choose any styling solution - plain CSS classes, a CSS framework, Emotion, etc.
-It is also possible to replace these default components by other HTML elements or custom components.
+## Basic switch
 
-There are three components you can override by the `components` prop: `Root`, `Thumb` and `Input`. Each one's props can be set using the `componentsProps` object.
+The `SwitchUnstyled` component is composed of three interior components—`Root`, `Thumb`, and `Input`—and it assigns CSS classes for styling each piece, respectively.
+
+You can set the props for these interior components using the `componentsProps` object.
+
+You can also override them entirely with the `components` prop, to replace them with other HTML elements or custom components.
+
+The following demo shows how to assign styles and props to the interior components of the `SwitchUnstyled` component:
 
 {{"demo": "UnstyledSwitches.js"}}
 
-## useSwitch
+## Accessibility
 
-For the ultimate customizability, a `useSwitch` hook is available.
-It accepts almost the same options as the SwitchUnstyled component minus the `component`, `components`, and `componentsProps` props.
+To make the `SwitchUnstyled` component accessible, you should ensure that the corresponding labels reflect the current state of the switch.
```

Comment:
Should we include a demo or a code snippet to elaborate on this?

Reply:
Maybe, it could be useful 👍